literature/literature.bib

```diff
@@ -9795,6 +10227,17 @@ @article{PTD+:EMSE16
 	year      = 2016,
 }
 
+@article{DBAM:ICEIS16,
+	author    = {da Silva, L{\'e}uson MP and Bezerra, Carla IM and Andrade, Rossana MC and Monteiro, Jos{\'e} Maria S},
```

Comment:
I am not 100% sure but I think we usually add a `.` behind name abbreviations (e.g., `M. P.` or `MP.`). Please cross-check.